hdr_html_canvas_element.md

```diff
@@ -411,3 +462,63 @@ Arguably, the HDR configuration data could be attached to the `CanvasRenderingCo
 
 The HDR configuration data should travel with an `ImageBitmap` when displayed in an `ImageBitmapRenderingContext`.
 That may inform where we should put this.
+
+### Are the new `ScreenAdvanced` attributes a fingerprinting vector?
+
+The [window placement proposal](https://github.com/webscreens/window-placement/blob/main/EXPLAINER.md) adds a mechanism through which detailed screen parameters may be queried, and changes in those parameters may be listened for.
+These parameters are a fingerprinting vector, and, as such, they are protected behind a permission prompt.
+
+### Should `ScreenAdvanced` expose HDR headroom or nit values?
+
+In this proposal, we expose a `highDynamicRangeHeadroom` parameter. One could imagine a different proposal with such information as:
+
+```idl
+  partial interface ScreenAdvanced {
+    // Minimum luminance that the screen can display, in nits.
```

Comment:
Is that the static minimum luminance, or the active minimum luminance with backlight dimming?

Reply:
Removed this section and put it inside https://github.com/w3c/ColorWeb-CG/issues/42